### ISSUE TITLE
What Info Theory do we need?

### ISSUE THREAD

**Issue by philschulz** [BasicProbability/BasicProbability.github.io#23]
I started with ch. 6. I now define entropy and show the binary entropy diagram. In addition, I will transfer the results we have for expectation (like the entropy of independent RVs is additive). I will finish by introducing the idea of KL divergence. Is there anything else you want me to put in?

**Comment by cschaffner** [comment]
Wow, chapter 5 is quite a mouthful! ;-) It looks pretty good (and long) to me.
See my changes in the pull request

**Comment by cschaffner** [comment]
I don't think we have to insert more information theory than what you think is necessary. There seems to be material enough.
